src/coreclr/nativeaot/Test.CoreLib/src/System/Runtime/RuntimeImports.cs

```diff
@@ -88,6 +88,9 @@ internal static unsafe Array RhNewArray(EETypePtr pEEType, int length)
         [DllImport(RuntimeLibrary)]
         internal static unsafe extern void RhAllocateNewObject(IntPtr pEEType, uint flags, void* pResult);
 
+        [DllImport(RuntimeLibrary)]
```

Comment:
Is this called from anywhere in Test.CoreLib?

Reply:
Nope, it is **not** called.
I was just following the pattern. I guess I can remove it if it is not called?

Reply:
Yup. We only put here those APIs that are needed.